tests/onnx/quantization/test_min_max.py

```diff
@@ -79,3 +82,13 @@ def test_get_channel_axes_deptwiseconv_node_ov(self):
 
     def test_get_channel_axes_matmul_torch(self):
         pytest.skip("Test is not applied for ONNX backend.")
+
+
+class TestONNXCommonMinMax(TemplateTestCommonMinMax, TestONNXMinMaxAlgorithm):
+    @staticmethod
+    def get_backend():
+        return BackendType.ONNX
+
+    @staticmethod
+    def get_no_quantized_ops_graph():
+        return NNCFGraphToTest(ONNXIdentityMetatype).nncf_graph
```

Comment:
Why used different metatypes as conv_metatype in NNCFGraphToTest?

And this tests looks like not backend specific that should check only cache. Could you patch functions test it without run backend specific code?

Reply:
made a test backend agnotic